lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -2295,8 +2315,15 @@ export class UserAgentApplication {
    * @param state
    */
   private setAccountCache(account: Account, state: string) {
+
     // Cache acquireTokenAccountKey
-    let accountId = account ? this.getAccountId(account) : Constants.no_account;
+    let accountId: string;
```

Comment:
Why are we expanding the ternary operators?

Reply:
This can be reverted. There are multiple iterations that this code went through during requirements clarity on where we want to put the check on client_info. Reverting in a new commit.